.github/workflows/commenting-on-old-issues.yml

```diff
@@ -0,0 +1,32 @@
+name: Commenting on old issues
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Every day at 00:00 UTC
+  workflow_dispatch:
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Comment on old issues
+        run: |
+          one_month_ago=$(date --date '1 month ago' '+%Y-%m-%d')
+
+          # For the search syntax, see https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests
+          old_issue_urls=$(gh issue list \
+            --search "is:open updated:<${one_month_ago} label:\"status: ask to implement\" -label:\"${label}\"" \
+            --json 'url' \
+            --jq '.[] | .url')
+
+          for url in ${old_issue_urls}; do
+            gh issue comment "${url}" --body "${issue_comment}"
+            gh issue edit "${url}" --add-label "${label}"
+          done
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          issue_comment: |-
+            This issue is older than 1 month. Even though it is labelled as "ask to implement", please ask before opening a pull request as it may no longer be relevant.
```

Comment:
nit

The `Even though it is labelled as "ask to implement",` part is unnecessary and contradictory with the rest of the sentence. Just `Please ask…" will do fine.

Reply:
Fixed with b01d660.